### ISSUE TITLE
Zero Validation Accuracy

### ISSUE THREAD

**Issue by hojjatb** [walsvid/VGG-TensorFlow#8]
duringrunning the code, zero validation accuracy always calculated. what's the problem?
